infra/cf-route53.yaml

```diff
@@ -23,23 +23,25 @@ Parameters:
     Type: String
     Description: The ID of the hosted zone
 
+Outputs:
+  DomainName:
+    Value: !Ref DomainName
+    Export:
+      Name: !Sub "DomainName-${Environment}"
```

Comment:
where is this output used in?

Reply:
Yes, it is used in the CF files which require domain name. There are 2 stacks created from this, 1 for staging and 1 for production. Each output contains the staging or production domain name, as defined in Github secrets.